app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchRandomGreeting } from "../redux/greetings";
+
+const Greeting = () => {
+  const dispatch = useDispatch();
```

Comment:
- [ ] Your app is not working as intended. Kindly ensure that _rack-cors_ gem is installed and properly configured.

Reply:
Hi @rloterh, the app works perfectly on my side. And for the a react-rails project the `rack-cors` gem may or may not be needed. 
To run the app, 
- In the project root - run `bundle install`
- run `npm install`
- run migrations - `rails db:migrate`
- seed database data - `rails db:seed`
- run the server in the terminal at port 3000 by typing `rails s`
- In another terminal, in the project root - run `./bin/webpack-dev-server`
- load app at `localhost:3000`

Please check the video below 👇🏻 


https://user-images.githubusercontent.com/4247740/156060635-0c46e31b-111f-44b6-9b08-7e81c5bd7e06.mp4

Reply:
Fellow Code Reviewer:

Kindly approve this project! This project is found to be complete upon further checking

Reply:
@Cathella 
Yes, your project is found complete upon checking further. I missed a very important step during the review. Kindly note that this was not a deliberate attempt.

Cheers! and Happy coding! 😃